src/drt/src/db/tech/frTechObject.h

```diff
@@ -32,32 +32,34 @@
 #include <iostream>
 #include <map>
 #include <memory>
+#include <set>
 #include <vector>
 
 #include "db/obj/frVia.h"
 #include "db/tech/frLayer.h"
 #include "db/tech/frViaRuleGenerate.h"
 #include "frBaseTypes.h"
 #include "utl/Logger.h"
-#include <set>
 namespace odb {
-  class dbTechLayer;
+class dbTechLayer;
 }
 namespace fr {
 namespace io {
 class Parser;
 }
 
-class frTechObject
-{
+class frTechObject {
```

Comment:
What version of clang-format are you using?  I don't see this happening when I run it (the { stays on a separate line).  This happens all over with different inline methods.

Reply:
@maliberty maybe using a bot like https://github.com/tinybeachthor/gitbot-format could make this easier.

Reply:
@maliberty I am using clang 7

Reply:
That's quite old - any chance you could upgrade?

Reply:
@maliberty is the formatting still off?